forms-flow-api/src/formsflow_api/utils/roles.py

```diff
@@ -0,0 +1,9 @@
+# """Role definitions."""
+
+
+# class Role:  # pylint: disable=too-few-public-methods
+#     """User Role."""
```

Comment:
looks like codes in this file are commented? are we using this file?. pls delete the file, if its not required